### DISCUSSION TITLE
项目显示未公开

### DISCUSSION THREAD

**Discussion by co63oc** [PaddlePaddle/PaddleOCR#12562] [Q&A]
https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.7/applications/%E6%89%AB%E6%8F%8F%E5%90%88%E5%90%8C%E5%85%B3%E9%94%AE%E4%BF%A1%E6%81%AF%E6%8F%90%E5%8F%96.md

![image](https://github.com/PaddlePaddle/PaddleOCR/assets/4617245/e457fb53-3a83-427d-960c-6c98cca86d82)
项目显示未公开
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/4617245/3c20ca9a-6b61-4546-9a52-99782c48273b)

**Comment by UserWangZz** [comment]
可能文档中链接没有更新过来，我反馈一下
